met-web/src/components/engagement/new/view/EngagementHero.tsx

```diff
@@ -41,36 +55,78 @@ export const EngagementHero = ({ engagement }: { engagement: Engagement }) => {
                     padding: { xs: '43px 16px 75px 16px', md: '88px 48px 88px 5vw', lg: '88px 48px 88px 156px' },
                     boxShadow:
                         '0px 20px 11px 0px rgba(0, 0, 0, 0.00), 0px 12px 10px 0px rgba(0, 0, 0, 0.01), 0px 7px 9px 0px rgba(0, 0, 0, 0.05), 0px 3px 6px 0px rgba(0, 0, 0, 0.09), 0px 1px 3px 0px rgba(0, 0, 0, 0.10)',
+                    alignSelf: { xs: 'flex-start', md: 'center' },
+                    alignContent: 'center',
                 }}
             >
-                <EyebrowText m="0">{engagement.sponsor_name}</EyebrowText>
-                <Grid container mt="16px" flexDirection="row" alignItems="center" columnSpacing={1}>
-                    <Grid item>
-                        <EngagementStatusChip statusId={engagement.submission_status} />
-                    </Grid>
-                    <Grid item>
-                        <BodyText bold size="small" sx={{ color: '#201F1E' }}>
-                            <time dateTime={`${startDate.format(semanticDateFormat)}`}>
-                                {startDate.format(dateFormat)}
-                            </time>{' '}
-                            to{' '}
-                            <time dateTime={`${endDate.format(semanticDateFormat)}`}>{endDate.format(dateFormat)}</time>
-                        </BodyText>
-                    </Grid>
-                </Grid>
-                <Header1 weight="thin" sx={{ color: colors.surface.gray[110], m: '40px 0' }}>
-                    {engagement.name}
-                </Header1>
-                <Button
-                    href={engagement.cta_url || '#cta-section'}
-                    variant="primary"
-                    size="large"
-                    icon={<FontAwesomeIcon fontSize={24} icon={faChevronRight} />}
-                    iconPosition="right"
-                    sx={{ borderRadius: '8px' }}
+                <Suspense
+                    fallback={
+                        <>
+                            <Skeleton>
+                                <EyebrowText m="0">Sponsor Name</EyebrowText>
+                            </Skeleton>
+                            <Skeleton>
+                                <Header1 weight="thin" sx={{ mb: '32px' }}>
+                                    Engagement Name
```

Comment:
You'll know best but is it possible that this fallback would be confusing if things take a long time to load, or never load successfully? It looks like the page structure would stay the same while loading, which is nice. But is it clear to folks that this is a loading view? Will it say "loading" or indicate as such anywhere on the page?